ui/tests/unit/routes/workspace/projects/project/settings-test.ts

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | workspace/projects/project/settings', function(hooks) {
```

Comment:
Seeing this made me wonder what tests for the new settings component might look like.